src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -1177,7 +1204,15 @@ private void verifyInventoryInteraction(CompositePurchaseOrder reqData, int crea
     // All existing and created items
     List<JsonObject> items = joinExistingAndNewItems();
 
-    assertEquals(createdPieces.size(), items.size());
+    int eQuantity0 = reqData.getCompositePoLines().get(0).getCost().getQuantityElectronic()!=null ? reqData.getCompositePoLines().get(0).getCost().getQuantityElectronic() : 0;
+    int physicalQuantity0 = reqData.getCompositePoLines().get(0).getCost().getQuantityPhysical()!=null ? reqData.getCompositePoLines().get(0).getCost().getQuantityPhysical() : 0;
+    int eQuantity1 = reqData.getCompositePoLines().get(1).getCost().getQuantityElectronic()!=null ? reqData.getCompositePoLines().get(1).getCost().getQuantityElectronic() : 0;
+    int physicalQuantity1 = reqData.getCompositePoLines().get(1).getCost().getQuantityPhysical()!=null ? reqData.getCompositePoLines().get(1).getCost().getQuantityPhysical() : 0;
+
+    int totalQuantity0 = eQuantity0 + physicalQuantity0 + eQuantity1 + physicalQuantity1;
```

Comment:
we could extract the total quantity calculation part to a separate method, as we are using in multiple places

Reply:
Done